gateways/adyen/adyen.go

```diff
@@ -59,7 +60,6 @@ func (client *AdyenClient) Authorize(request *sleet.AuthorizationRequest) (*slee
 	response := &sleet.AuthorizationResponse{
 		TransactionReference: result.PspReference,
 	}
-
 	if result.AdditionalData != nil {
```

Comment:
Do you think it makes sense to introduce a new function that adds the additional data fields? This if-clause is getting pretty long and making the `Authorize` function pretty long too.

Reply:
yep i'll break it out